addon/object/computed.js

```diff
@@ -1,10 +1,11 @@
 import Ember from 'ember';
-
 import {
   decoratedPropertyWithRequiredParams,
   decoratedPropertyWithOptionalCallback
 } from '../utils/decorator-macros';
 
+const { assert } = Ember;
```

Comment:
My only concern with module imports is backwards compat, we're using `ember-decorators` on a fairly old version of Ember/ember-cli-shims